concordium-node/src/bin/cli.rs

```diff
@@ -138,10 +140,49 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
-    if let Some(ref import_path) = conf.cli.baker.import_path {
-        info!("Starting out of band catch-up");
-        consensus.import_blocks(import_path.as_bytes());
-        info!("Completed out of band catch-up");
+    // Out-of-band catch-up
+    if let Some(ref import_blocks_from) = conf.cli.baker.import_blocks_from {
```

Comment:
This should probably be factored out in a separate function, possibly on `ConsensusContainer`. Errors should be handled gracefully, and we probably shouldn't stop the node if we can recover, even if download/import was not successful.

Reply:
Not sure if the downloader function should be part of `ConsensusContainer` but I factored it out for the sake of returning a `Result` instead of stopping the node.